index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var Server = require('./lib/server');
+
+module.exports = {
+  server: function (opts) { return new Server(opts); },
```

Comment:
Should we expose the Server and Worker classes as opposed to this factory method?